### PR TITLE
Add stop_server command to Makfile

### PR DESCRIPTION
**Pull request by khashf**
To stop pelican process created by `make serve` and `make devserve`. `osl-pelican` has this command but this repo doesn't.

### CODE REVIEW

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://beaver-barcamp-pelican-53.staging.osuosl.org

File: Makefile [hunk 1]
```diff
@@ -55,6 +56,11 @@ serve:
 devserver:
 	$(BASEDIR)/develop_server.sh restart
 
+stopserver:
+	kill -9 `cat pelican.pid`
+	kill -9 `cat srv.pid`
```
Comment (ramereth) [inline Makefile]: Do we need to use ``-9``?

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://beaver-barcamp-pelican-53.staging.osuosl.org

File: Makefile [hunk 2]
```diff
@@ -29,6 +29,7 @@ help:
 	@echo '   make publish                     generate using production settings '
 	@echo '   make serve [PORT=8000]           serve site at http://localhost:8000'
 	@echo '   make devserver [PORT=8000]       start/restart develop_server.sh    '
+	@echo '   make stopserver                  stop local server                  '
```
Comment (ramereth) [inline Makefile]:
```suggestion
	@echo '   make stopdevserver                  stop dev server                  '
```

File: Makefile (hunk 1 above)
Comment (khashf) [reply Makefile]: Not necessary. I've revised here and in osuosl-pelican ([PR](https://github.com/osuosl/osuosl-pelican/pull/257)), too

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://beaver-barcamp-pelican-53.staging.osuosl.org

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://beaver-barcamp-pelican-53.staging.osuosl.org

**Comment by khashf** [comment]
[Add BarCamp video to Pelican](https://trello.com/c/7V8QEgWA/1270-add-barcamp-video-to-pelican)

**Pull request merged**